### ISSUE TITLE
make links in internet lobby clickable

### ISSUE THREAD

**Issue by bunnybot** [widelands/widelands#2054]
If one accesses the Metaserver lobby there is a note:

*** For hosting a game, please take a look at the notes at:
*** http://wl.widelands.org/wiki/InternetGaming

It would be very nice to have this link clickable so that one doesn't have to enter the whole name into a browser.

------------------------------------
Imported from Launchpad using lp2gh.

 * date created: 2016-06-26T19:23:37Z
 * owner: franku
 * the launchpad url was https://bugs.launchpad.net/bugs/1596349

**Comment by bunnybot** [comment]
(by sirver)
The new font renderer supports clickable links - in theory. Once we do not use the old one anywhere anymore, we can implement that using the new one.
